osc_sdk/sdk.py

```diff
@@ -126,6 +134,25 @@ def __init__(self, **kwargs):
         self.date = None
         self.datestamp = None
 
+    def setup_cmd_options(self, login, password, authentication_method):
+        self.authentication_method = authentication_method
+        self.login = login
+        self.password = password
+
+    def check_options(self):
```

Comment:
I know it's not working but I find it sad, that we require auth to use call that doesn't require one (like ReadRegions)

Reply:
Probably to fix as a separate PR